README.md

```diff
@@ -48,7 +49,14 @@ interactive differential equations. For example, here is an exponential function
 computed as the solution of a differential equation:
 
 ```purescript
-exp = fixB 1.0 \b -> integrate 1.0 time ((-2.0 * _) <$> b)
+exp = fixB 1.0 \b -> integral' 1.0 seconds ((-2.0 * _) <$> b)
 ```
+Its differential equation is   <a href="https://www.codecogs.com/eqnedit.php?latex=\inline&space;\large&space;\frac{\partial&space;y}{\partial&space;t}&space;=&space;-2y&space;\\" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\inline&space;\large&space;\frac{\partial&space;y}{\partial&space;t}&space;=&space;-2y&space;\\" title="\large \frac{\partial y}{\partial t} = -2y \\" /></a> ,  which is rearranged so the right hand side gives an expression whose fixed point solution is the exponential function desired.   
```

Comment:
Can this use markdown syntax please?

Reply:
In fact, I don't think latex is even needed here. A unicode integral sign would work just as well, I think.

Reply:
E.g. 

```
𝑑𝑥/𝑑𝑡 = -2𝑥
𝑥 = ∫-2𝑥 𝑑𝑡
```